packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-attach/notebook-attach.controller.js

```diff
@@ -96,19 +96,17 @@ export default class NotebookAttachController {
 
   filterStorages() {
     this.filteredStorages = this.storages
-      // Remove containers that are already on volume list
+      // Remove containers that are already on volume list and and which are isHighPerfStorage.
       .filter(({ name, region }) => {
         return !this.notebookModel.volumes
           .filter(({ privateSwift }) => privateSwift)
           .map(({ container }) => `${container.name}-${container.region}`)
           .includes(`${name}-${region}`);
       })
-      .map(({ name, region }) => {
-        return {
-          name,
-          region,
-          description: `${name} - ${region}`,
-        };
+      .flatMap(({ name, region, isHighPerfStorage }) => {
+        return !isHighPerfStorage
+          ? { name, region, description: `${name} - ${region}` }
+          : null;
```

Comment:
Don't you need to filter those null values after?

Reply:
With the flatMap the null values do not go back into the table.

Reply:
Indeed I made a mistake, it is [ ] and not null that I need.
I have applied the change.